src/org/mozilla/javascript/SlotMap.java

```diff
@@ -25,21 +25,21 @@
 
     /**
      * Return the Slot that matches EITHER "key" or "index". (It will use "key" if it is not null,
-     * and otherwise "index". "accessType" is one of the constants defined in ScriptableObject.
+     * and otherwise "index".) If no slot exists, then create a default slot class.
```

Comment:
If we touching the javadoc here what about using @retruns and @param.

Reply:
Thanks -- I did a little bit of cleanup.